mirrord/layer/src/file/filter.rs

```diff
@@ -58,9 +58,10 @@ fn generate_local_set() -> RegexSet {
         r"^/dev/.*$",
         r"^/opt/.*$",
         r"^/home/.*$",
-        r"^/tmp/.*$",
-        r"^/private/.*$",
-        r"^/var/folders/.*$",
+        // don't use `/.*$` so lstat on the directory will also be local
+        r"^/private",
+        r"^/var/folders",
+        r"^/tmp",
```

Comment:
```suggestion
        r"^/private(/|$)",
        r"^/var/folders(/|$)",
        r"^/tmp(/|$)",
```
If we want to match `/private`, `/private/` and `/private/foo`, but not `/private_key`

Reply:
gee I'm bad at regex, thanks!